azext_iot/operations/hub.py

```diff
@@ -1334,7 +1334,7 @@ def iot_device_upload_file(cmd, device_id, file_path, content_type, hub_name=Non
 def iot_hub_monitor_events(cmd, hub_name=None, device_id=None, consumer_group='$Default', timeout=300,
                            enqueued_time=None, resource_group_name=None, yes=False, properties=None, repair=False,
                            login=None, content_type=None, device_query=None):
-    _iot_hub_monitor_events(cmd, interface=None, pnp_context=None, hub_name=hub_name, device_id=device_id,
+    _iot_hub_monitor_events(cmd, interface_name=None, pnp_context=None, hub_name=hub_name, device_id=device_id,
```

Comment:
We should simply delete `interface` + `pnp_context` arguments from the call to `_iot_hub_monitor_events` here since DT/pnp stuff is unrelated to general IoT hub monitoring and the default values are None anyway.

Reply:
Fixed